Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

```diff
@@ -26,16 +26,51 @@ When an instance of Umbraco starts up it generates some 'temporary' files on dis
 
 ### Umbraco PublishedCache
 
-When Azure Web Apps auto transitions between hosts, you scale the instances or you utilise slot swapping you may experience issues with the Umbraco Published Cache becoming locked unless Umbraco is configured to use SQL for MainDom locking. 
+When Azure Web Apps auto transitions between hosts, you scale the instances or you utilise slot swapping you may experience issues with the Umbraco Published Cache becoming locked unless Umbraco is configured correctly.
+
+#### Backoffice
+
+For the backoffice Web App **only** you should configure Umbraco to use SQL for MainDom locking. 
```

Comment:
Do we define 'MainDom locking' anywhere else in the docs.. just to be clear on what the mechnism is for, makes it easier to understand why it should be configured correctly